documentation/2_Guiding_Principles.md

```diff
@@ -396,3 +396,72 @@ Such roles originate from regulations, like GDPR or DGA:
 * Marketplace and other value creation services are covered in the IDSA Rulebook above and in the optional capabilites section of the functional requirements.
+
+# Understanding Roles and Layers in Data Spaces
+
+Data spaces are multi-layered ecosystems that rely on the seamless integration of technical protocols, business processes, and legal frameworks. One of the foundational challenges in governing data spaces lies in the consistent definition and use of key concepts such as **"roles," "policies,"** and **"contracts."** These terms often carry different meanings across domains. This chapter establishes a clear separation between technical and non-technical interpretations of such concepts to support the development of interoperable and trustworthy data spaces.
+
+## Layers of a Data Space
+
+Data spaces can be structured into three primary layers, each serving distinct functions:
+
+1. **Technical Layer** – Encompasses the architecture and protocols (e.g., Data Space Protocol (DSP), Decentralized Claims Protocol (DCP)) that facilitate secure and interoperable data exchanges.
+2. **Business/Operational Layer** – Manages the services, interactions, and workflows that enable value generation and marketplace activity.
+3. **Legal and Governance Layer** – Enforces rights, obligations, and regulatory compliance across participants.
+
+
+![Layers of Data Spaces](./media/media/image20.png)
+
+These layers interact but must be conceptually separated to ensure clarity and reduce ambiguity in roles and responsibilities.
+
+## Clarifying the Concept of Roles
+
+The term **"role"** is context-dependent and must be clearly scoped:
+
+- At the **technical level**, there is only one fundamental role: **participant**. A participant acts as a **data provider**, a **data consumer**, or both within the data exchange protocol.
+- At the **business level**, participants may take roles such as **data intermediary**, **marketplace operator**, **auditor**, or **service provider**.
+- These business roles do not exist independently at the technical layer but are mapped onto the core participant role based on the services performed.
+
+Maintaining this distinction ensures that governance models remain technically sound while accommodating diverse business scenarios.
+
+## Distinguishing Data Spaces from Trusted Data Transactions
+
+A clear differentiation must be made between **data spaces** and **trusted data transactions (TDTs)**:
+
+- **Data spaces** are decentralized infrastructures that enable sovereign data exchange based on open standards. They preserve participant autonomy and operate without mandatory intermediaries.
+- **Trusted data transactions**, often associated with the *Data Governance Act* and specifications from initiatives such as *Gaia-X*, emphasize **data intermediaries** and **service orchestration**. These models prioritize regulatory alignment and controlled environments.
```

Comment:
We should avoid referencing the _data governance act_, _Gaia-X_, etc. as this is very European focused. While for us it is clear, that is definitely not true from an international audience perspective. Please at least mention it as _"European flavor"_ or better only reference formal standardization activities (here CEN JTC25).